wgpu-hal/src/metal/device.rs

```diff
@@ -8,6 +8,7 @@ use std::{
 
 use super::conv;
 use crate::auxil::map_naga_stage;
+use metal::foreign_types::ForeignTypeRef;
```

Comment:
opted to remove the direct dependency to foreign-types since seems to originate deeper down the tree than Metal. Isn't used for anything else than this here anyways